src/pool/pool_jemalloc.c

```diff
@@ -459,6 +462,13 @@ static umf_result_t op_initialize(umf_memory_provider_handle_t provider,
     if (!pool) {
         return UMF_RESULT_ERROR_OUT_OF_HOST_MEMORY;
     }
+    memset(pool, 0, sizeof(*pool) + n_arenas * sizeof(*pool->arena_index));
+    const char *pool_name = DEFAULT_NAME;
+    if (jemalloc_params) {
+        pool_name = jemalloc_params->name;
+    }
+    strncpy(pool->name, pool_name, sizeof(pool->name) - 1);
```

Comment:
please be consistent - use sprintf everywhere

Reply:
done